### ISSUE TITLE
Changing FTP should not affect previous fitness score, only new calculated fitness score?

### ISSUE THREAD

**Issue by klavan88** [thomaschampagne/elevate#332]
**Bug description:** 
Not really a bug, but a change of the FTP changes the fitness score, it should only affect new activities from that date? My FTP was not the same 2 months ago, so the fitness score of 2 months ago, should be calculated against the active FTP value from 2 months ago, not the new value.

**Actual Behavior:** 
Changing the FTP changes the fitness score, it should only affect new activities from that date? My FTP was not the same 2 months ago, so the fitness score of 2 months ago, should be calculated against the active FTP value from 2 months ago, not the new value.

**Expected Behavior:** 
FTP change should only affect new activities? 

**Steps to Reproduce:** 
- Change FTP value and do a full resync
 - ...
 - ...

**Chrome version** 
Version 56.0.2924.87 (64-bit)

**Plugin version:** 
5.4.2

**Activities links?:** 

**Console errors? (press F12 to see developer console, and copy paste here):** 

``` 
put console errors here if exist 
``` 

**Link screenshots or youtube video link if necessary:**

**Comment by snowfree52** [comment]
same problem here, without that, the fitness trend is a bit useless :(

**Comment by tallpaulhunter** [comment]
I'm seeing this problem too.
Maybe a table needs to be incorporated in Athlete settings utilising date and FTP test value.
For example:
At the end of 2017 I had a period off the bike. Before starting to train I performed a test to find my starting FTP value for 2018 (01/JAN/2018 = 224).
At the start of February I recorded an FTP increase (02/FEB/2018 = 243).
At the end of March I perform a further FTP test (26/MAR/2018 = 296).

The fitness score for 01/JAN/2018 to 01/FEB/2018 should use an FTP value of 224.
Between 02/FEB/2018 and 25/MAR/2018 a value of 243 should be used in calculations.
From 26/MAR/2018 a value of 296 should be used.
